pkg/virtualKubelet/forge/meta_test.go

```diff
@@ -76,6 +71,48 @@ var _ = Describe("Meta forging", func() {
 		}))
 	})
 
+	Describe("the RemoteObjectMeta function", func() {
```

Comment:
Do you think having a test that checks for the local object to be unchanged would be useful?